docs/tarefa2.md

```diff
@@ -14,10 +26,10 @@ Agora é hora de encontrar uma issue (tarefa ou problema) para trabalhar. Esse 
 Siga estas etapas simples para encontrar sua primeira issue:
 
 1. Acesse o Repositório: Navegue até a página principal do projeto no GitHub: <https://github.com/cumbucadev/gitcaos>
-1. Na parte superior da página do repositório principal, você verá várias abas. Clique na aba chamada Issues. Aqui, você encontrará uma lista de tarefas disponíveis que precisam de ajuda.
-1. Procure pela issue específica: Agora, você precisa encontrar uma issue com o título: # Tarefa 2: O Jogo do Tigrinho me deu (seu-username). Essa tarefa foi criada especialmente para você, e é uma ótima oportunidade para praticar.
-1. Verifique se a issue já está designada para alguém: Olhe ao lado do título da issue para ver se ela já tem alguém designado (chamado de assignee). Se o seu nome já estiver lá, ótimo! Isso significa que você está pronto para trabalhar na tarefa. Se não estiver, você pode pedir para ser designado. Para isso, deixe um comentário na issue dizendo algo como: "Oi, eu gostaria de trabalhar nessa tarefa! Você pode me designar?". Isso garante que nunca mais de uma pessoa trabalhe na mesma issue, evitando conflitos e confusões no desenvolvimento.
-1. Leia atentamente as instruções: Antes de começar a trabalhar, é fundamental ler todas as instruções que estão na issue. Isso vai garantir que você entenda o que é solicitado e que siga as diretrizes corretamente. Preste atenção a detalhes como formato de arquivo, informações que devem ser incluídas e qualquer outro requisito.
+2. Na parte superior da página do repositório principal, você verá várias abas. Clique na aba chamada Issues. Aqui, você encontrará uma lista de tarefas disponíveis que precisam de ajuda. 
```

Comment:
O lance de colocar tudo 1. é que o markdown enumera para você automaticamente. Aí vc não precisa atualizar a numeração toda vez que altera a ordem ou adiciona/remove um item:

https://www.seancdavis.com/posts/pro-tip-use-1-for-every-ordered-list-item-when-writing-markdown/

Então nesse caso, seria legal manter tudo `1.` mesmo